### DISCUSSION TITLE
How to show modal from v0.26.x

### DISCUSSION THREAD

**Discussion by jongbonga** [cdmoro/bootstrap-vue-3#2377] [Q&A] [answered]
I updated to the latest version and my modals are not triggering anymore using `const {show} = useModal("modal-id")`

**Comment by xvaara** [comment]
can you share your code?

works in this example:
https://stackblitz.com/edit/github-ydzoyh-yoz2nz?file=src%2FApp.vue

**Reply by jongbonga** [reply]
@xvaara after further testing, the issue only happens when the modal has the `lazy` prop.

See [here](https://stackblitz.com/edit/github-ydzoyh-nl1uhp?file=src%2FApp.vue)

**Reply by xvaara** [reply]
I created an issue of it, not sure when someone has the time to look into it.

**Reply by xvaara** [reply, marked as answer]
should be fixed in the latest release
